packages/jss/src/plugins/keyframesRule.js

```diff
@@ -79,6 +79,23 @@ export class KeyframesRule implements ContainerRule {
 
 const keyRegExp = /@keyframes\s+/
 
+/**
+ * Replace the reference for a animation name.
+ */
+function replaceRef(style, prop, keyframes) {
```

Comment:
I recently stopped using function declarations, for the same reasons as we don't use "var" + hoisting which allows writing inconsistent code with function after usage.

Reply:
https://twitter.com/oleg008/status/1052133439980363777

Reply:
I don't know, eslint should cache such bugs and I think the function declaration syntax is a bit nicer than using `const` with arrow functions

Reply:
Yeah, bugs is not much of a problem with eslint, its more consistency to me.

Reply:
I did some changes, will push in a minute